Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -338,7 +349,7 @@ - (void)_removeVisibilityPropagationViewForWebProcess
 {
 #if USE(EXTENSIONKIT)
     if (_visibilityPropagationInteractionForWebProcess) {
-        [self removeInteraction:_visibilityPropagationInteractionForWebProcess.get()];
+        [_visibilityPropagationViewForWebProcess removeInteraction:_visibilityPropagationInteractionForWebProcess.get()];
```

Comment:
Do we need to remove the view from the parent, too?

Reply:
Good point! That is done in the code below.

Thanks for reviewing!